app/views/state_file/questions/nj_eligibility_health_insurance/edit.html.erb

```diff
@@ -17,7 +17,6 @@
                   label: t("general.negative"),
                   input_html: {
                     "data-follow-up": "#household-members",
-                    "aria-controls": "household-members" 
```

Comment:
[dust] I also don't understand the removal of this -- was it not effective or correct as it was?

Reply:
Axe flagged it, so I think it was either not correct, or needs to be added with js when the associated element appears, or was a problem bc of nested fieldsets. Going to surface this with Damian after the fieldset PR lands